gitrevise/tui.py

```diff
@@ -188,9 +188,9 @@ def noninteractive(
 def inner_main(args: Namespace, repo: Repository) -> None:
     # If '-a' or '-p' was specified, stage changes.
     if args.all:
-        repo.git("add", "-u")
+        repo.git("add", "-u", stdout=DEVNULL)
     if args.patch:
-        repo.git("add", "-p")
+        repo.git("add", "-p", stdout=DEVNULL)
```

Comment:
I think you actually uncovered a bug here, as `git add -p` isn't very useful if the output is captured. We should probably not capture the output of this command at all.

Reply:
I noticed that too but intended to keep this set of changes pure since that was an existing bug. Will push up a patch for this.

I suspect that it _might_ actually work (at least partially) if `add -p` and `-i` use `/dev/tty` instead of `/dev/stdout`, which could be the reason it was overlooked.

Reply:
That's pushed up